### PR TITLE
Add new capabilities for AU Becs Debit and tax reporting

### PR DESCRIPTION
**Pull request by remi-stripe**
cc @stripe/api-libraries

### CODE REVIEW

File: account.go [hunk 1]
```diff
@@ -330,9 +333,12 @@ type AccountBusinessProfile struct {
...
 // AccountCapabilities is the resource representing the capabilities enabled on that account.
 type AccountCapabilities struct {
-	CardPayments   AccountCapabilityStatus `json:"card_payments"`
-	LegacyPayments AccountCapabilityStatus `json:"legacy_payments"`
-	Transfers      AccountCapabilityStatus `json:"transfers"`
+	AUBECSDebitPayments    AccountCapabilityStatus `json:"au_becs_debit_payments"`
+	CardPayments           AccountCapabilityStatus `json:"card_payments"`
+	LegacyPayments         AccountCapabilityStatus `json:"legacy_payments"`
+	TaxReportingUS1099K    AccountCapabilityStatus `json:"tax_reporting_us_1099_k"`
+	TaxReportingUS1099MISC AccountCapabilityStatus `json:"tax_reporting_us_1099_misc"`
```
Comment (ob-stripe) [inline account.go]:
```suggestion
	TaxReportingUS1099Misc AccountCapabilityStatus `json:"tax_reporting_us_1099_misc"`
```

Comment (remi-stripe) [reply account.go]: Why? Isn't MISC an acronym?

Comment (ob-stripe) [reply account.go]: I think it's just an abbreviation for "miscellaneous"?

Comment (ob-stripe) [reply account.go]: Hm, but the 1099 form is called "Form 1099-MISC". So your version is probably fine.

**Pull request merged**